content/admin/configuration/configuring-network-settings/enabling-subdomain-isolation.md

```diff
@@ -20,7 +20,7 @@ topics:
 
 Subdomain isolation mitigates cross-site scripting and other related vulnerabilities. For more information, see "[Cross-site scripting](http://en.wikipedia.org/wiki/Cross-site_scripting)" on Wikipedia. We highly recommend that you enable subdomain isolation on {% data variables.product.product_location %}.
 
-When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains.
+When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains and references to user-supplied content by path will result in a 404 error.
```

Comment:
Thanks for this contribution, @ppottorff!

I briefly tested this on a GHES instance, and it wasn't all paths that would 404. For example, images in `http(s)://HOSTNAME/media/` looks like are redirected to the new path. Content  in `http(s)://HOSTNAME/raw/` doesn't get redirects, so I've adjusted your wording to mention that some URLs might `404`.

```suggestion
When subdomain isolation is enabled, {% data variables.product.prodname_ghe_server %} replaces several paths with subdomains. After enabling subdomain isolation, attempts to access the previous paths for some user-supplied content, such as `http(s)://HOSTNAME/raw/`, may return `404` errors .
```